gravitee-am-service/src/main/java/io/gravitee/am/service/model/PatchDomain.java

```diff
@@ -260,7 +267,7 @@ public Set<Permission> getRequiredPermissions() {
             requiredPermissions.add(Permission.DOMAIN_SETTINGS);
         }
 
-        if(alertEnabled != null && alertEnabled.isPresent()) {
+        if (alertEnabled != null && alertEnabled.isPresent()) {
```

Comment:
**question**: are we sure that this change will not break the current behaviour ? :thinking: This change will probably allow to disable the Alert feature een if the user doesn't have the DOMAIN_ALERT permission. I think this change should be revert. (I can see that we test the `isPresent` in the previous statement but don't know if we have to apply the same for alerts)

Reply:
It has been added only a space to fix format here.

Reply:
:tired_face: sorry about that one